packages/wrapped-keys-lit-actions/src/lib/common/internal/getDecryptedKey.js

```diff
@@ -0,0 +1,20 @@
+/* global Lit */
+
+export async function getDecryptedKey({
+  accessControlConditions,
+  ciphertext,
+  dataToEncryptHash,
+}) {
+  try {
+    // May be undefined, since we're using `decryptToSingleNode`
```

Comment:
This should always return the decrypted data to the selected node, what is meant by this comment?

Reply:
This was the existing logic, which i retained here.  My understanding is that because this code runs on all of the nodes, but the `decryptToSingleNode` call only actually returns a value on  _the one node that is chosen by backend consensus_, all of the the nodes that are executing this action that _aren't_ chosen to be 'the single node' that runs the decrypt will return undefined for this call.  As a result, we can't continue executing any further logic on that node since we don't have the privateKey in scope -- so  we exit immediately, and that node returns an empty result.  The SDK looks at the returned results from all nodes, and uses the only _non-empty result_ from the responses from all of the nodes.